tests/tracer/test_encoders.py

```diff
@@ -398,6 +398,36 @@ def test_span_types(encoding, span, tags):
     assert decode(refencoder.encode_traces([trace])) == decode(encoder.encode())
 
 
+def test_span_link_v05_encoding():
+    encoder = MSGPACK_ENCODERS["v0.5"](1 << 20, 1 << 20)
+
+    span = Span("name")
+
+    span._set_span_link(
```

Comment:
should this use `span.link_span`?

Reply:
I was avoiding creating two spans here. `link_span` is tested in `test_span`. I just wanted a minimal test that asserts that the span link fields were encoded correctly. I can stick to using the public here if that's preferable.

Reply:
probably better to focus on using the public interface wherever possible. starts to make the feasibility to move internals around harder later if they are littered all over the tests.

observation, not a strong opinion.